interpreter/k8s/src/main/scala/io/buoyant/transformer/k8s/LocalNodeTransformerInitializer.scala

```diff
@@ -0,0 +1,20 @@
+package io.buoyant.transformer.k8s
+
+import io.buoyant.namer.{NameTreeTransformer, TransformerConfig, TransformerInitializer}
+import java.net.InetAddress
+
+class LocalNodeTransformerInitializer extends TransformerInitializer {
+  val configClass = classOf[LocalNodeTransformerConfig]
+  override val configId = "io.l5d.localnode"
```

Comment:
similarly, if this is k8s-oriented, would like to have _k8s_ somewhere in the plugin name.